src/api/movieApi.ts

```diff
@@ -0,0 +1,12 @@
+const url = 'https://www.omdbapi.com/?apikey=e63cfb8c&t=';
+
+export const getMovie = (title: string) => {
+  return fetch(`${url}${title}`)
```

Comment:
unless a library or legacy codebase forces you to use then/catch, the better choice for readability and maintainability is async/await